packages/e2e-tests/specs/widgets/editing-widgets.test.js

```diff
@@ -420,6 +425,8 @@ describe( 'Widgets screen', () => {
 				if ( existingMarqueeWidgets ) {
 					await existingMarqueeWidgets.focus();
 					await pressKeyWithModifier( 'access', 'z' );
+					await closedPanelBody.focus();
+					await closedPanelBody.click();
```

Comment:
I think this is not required anymore ...

Reply:
Even if we revert this change, I believe the test should pass 🙂

Reply:
On it